aio/content/marketing/index.html

```diff
@@ -100,7 +100,7 @@ <h1 class="no-anchor no-toc" style="display: none"></h1>
 
     <!-- CTA CARDS -->
     <div layout="row" layout-xs="column" class="home-row">
-      <a href="start">
+      <a href="start" class="card-wrapper">
```

Comment:
Rather than add another class here, I think it actually makes the most sense to use the `<a>` as the card element.

This eliminates the odd case of padding being present within the `<a>` from the card but ending up as clickable area.

```suggestion
      <a href="start" class="card">
        <img src="generated/images/marketing/home/code-icon.svg" alt="Get Started with Angular" width="70" height="70" loading="lazy">
        <div class="card-text-container">
          <div class="text-headline">Try it now</div>
          <p>Explore Angular's capabilities with a ready-made sample app. No setup required.</p>
        </div>
      </a>
```


Alternatively, we could just switch the `<a>` and `<div class="card">` in the hierarchy and perform a similar result.

Reply:
yeah, you're totally right! I made the change :slightly_smiling_face::+1: